app/selectors.js

```diff
@@ -239,8 +239,17 @@ export const hasUnminedTransactions = compose(l => l && l.length > 0, get(["grpc
 export const transactions = createSelector(
   [transactionsNormalizer, get(["grpc", "transactions"])], apply
 );
+
 export const homeHistoryTransactions = createSelector(
-  [transactionsNormalizer, get(["grpc", "recentTransactions"])], apply
+  [transactions],
+  (transactions) =>
+    transactions.map(tx => {if (!tx.txType || tx.txType == "Regular") return tx; }).filter(tx => tx !== undefined)
+);
+
+export const homeHistoryTickets = createSelector(
+  [transactions],
```

Comment:
Same thing applies as previous comment. Either need to use `grpc.recentTransactions` or more likely (if we want to always fill with the 5 most recent stake txs) we'll need a separate state to track the home tickets.